README.md

```diff
@@ -134,17 +134,41 @@ Redis is a data structures server so unlike in plain-key value stores in which s
 
 ## Installing the Module ##
 
-First clone this respository if you haven't already anywhere on your local system. To do this, first go into the directory you wish to clone the repository in and type into the command line "git clone https://github.com/cmsc22000-project-2018/redis-tries.git"
+First clone this respository if you haven't already anywhere on your local system. To do this, first go into the directory you wish to clone the repository and then type into the command line: 
 
-Inside the *module* directory, type "make all" into the command line to build the trie.so library.
+    git clone https://github.com/cmsc22000-project-2018/redis-tries.git
+
+Inside the repository, initialize the submodule by typing into the command line:
+
+    git submodule update --init --recursive
+
+Inside the *module* directory, type "make all" into the command line to build the trie.so library. (Note: The compiler warnings can be ignored.)
 
 Print the working directory and write it down, appending "trie.so" to the end of the path. So if my name is Dustin and my redis-tries directory is on my Desktop, I should have written down /Users/Dustin/Desktop/redis-tries/module/trie.so
 
 Before you begin running the Redis server, make sure to check the src directory in redis-stable for a file called "dump.rdb" if it exists, make sure to delete it before running the server every time you decide to run the server. This is because we have not implemented the data persistence functions.
 
-Assuming you have Redis fully installed and functional (instructions here if you don't: https://github.com/cmsc22000-project-2018/redis-tries/wiki/Design-Document-v1), run the Redis server with "redis-server" in the redis-stable directory.
+At this point, we will assume you have Redis fully installed and functional. (Instructions are here if you don't: https://github.com/cmsc22000-project-2018/redis-tries/wiki/Design-Document-v1.)
+
+Inside the redis-stable directory, run the Redis server by typing: 
+
+    redis-server
+
+At the end of every session, close the server with CTRL+C.
+
+<pre>
```

Comment:
This shouldn't be here.

Reply:
Do you mean the <pre>?

Reply:
If so, I put it there it block out the note, so that readers would be able to distinguish it from the redis-cli command underneath while still keeping it next to the redis-server command.